src/modules/billing/mappers/transaction.js

```diff
@@ -148,13 +148,16 @@ const dbToModel = row => {
   transaction.fromHash({
     id: row.billingTransactionId,
     ...pick(row, ['status', 'isCredit', 'authorisedDays', 'billableDays', 'description', 'transactionKey',
-      'externalId', 'calculatedVolume', 'twoPartTariffError', 'twoPartTariffStatus']),
+      'externalId', 'calculatedVolume', 'twoPartTariffStatus']),
     chargePeriod: new DateRange(row.startDate, row.endDate),
     isCompensationCharge: row.chargeType === 'compensation',
     chargeElement: chargeElementMapper.dbToModel(row.chargeElement),
     volume: parseFloat(row.volume),
     agreements: mapDBToAgreements(row),
-    twoPartTariffReview: mapReviewDataToUser(row.twoPartTariffReview)
+    twoPartTariffReview: mapReviewDataToUser(row.twoPartTariffReview),
+    ...pickBy({
+      twoPartTariffError: row.twoPartTariffError
+    }, isBoolean)
```

Comment:
Note we added an assertIsNullableBoolean to the transaction model for twoPartTariffError so no need to make the change here however this is a belt and braces approach so could probably stay as is.